### ISSUE TITLE
phpLiteAdmin is Not compatible to php 5.3

### ISSUE THREAD

**Issue by GaneshKandu** [phpLiteAdmin/pla#18]
Hi,

just installed phpLiteAdmin 1.9.8.1 got this error
```
Array
(
    [type] => 1
    [message] => Call to undefined function getimagesizefromstring()
    [file] => phpliteadmin.php
    [line] => 1601
)
```
this function needs PHP 5 >= 5.4.0, PHP 7

**Comment by GaneshKandu** [comment]
steps to reproduce
create table with column blob type in any database and insert image and try to download or view
on php 5.3

**Comment by crazy4chrissi** [comment]
Thank you for reporting. The problem is, as you may guess, we don't really test phpLiteAdmin with these old PHP versions...
So I just fixed this. Could you please try the development version here:
https://www.phpliteadmin.org/phpliteadmin-dev.zip

**Comment by crazy4chrissi** [comment]
The commit fixing it was a263a8c6f1454603276b76a73cf817e281608bd0

**Comment by GaneshKandu** [comment]
Thanks @crazy4chrissi worked for me

**Comment by crazy4chrissi** [comment]
Fixed in phpLiteAdmin 1.9.8.2 released just now
